examples/gpu/README.md

```diff
@@ -0,0 +1,75 @@
+<!--
+   Copyright 2022, UNSW
+   SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+# Gpu example
+
+This example does a "happy path" testing of the gpu subsystem. It showcases all
+gpu requests and the most common state transitions that can occur.
+
+## Building
+
+The following platforms are supported:
+* qemu_virt_aarch64
+
+Note that this example depends on `imagemagick`.
+
+For `apt` users: `sudo apt-get install imagemagick`.
+
+For Homebrew users: `brew install imagemagick`.
+
+### Simulating on QEMU
+This example only supports simulating with QEMU only linux machines as it requires the 'udmabuf' framework.
+This is a requirement in order to enable virtio-gpu blob resources. Additionally, sudo permissions is required
+to access the `udmabuf` framework.
+
+This example also requires you to have a compatible GUI installed for QEMU.
+Make sure QEMU's default display is not set to `-display none`.
+
+### Make
+
+```sh
+make MICROKIT_SDK=<path/to/sdk> MICROKIT_BOARD=<board>
```

Comment:
As this supports only qemu, you can add the `MICROKIT_BOARD` in the Makefile, and not need it on the command line.